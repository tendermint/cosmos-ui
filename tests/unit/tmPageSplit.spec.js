import { shallowMount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import { TmPageSplit } from "@/index"

describe("TmPageSplit.vue", () => {
  let wrapper
  beforeEach(async () => {
    wrapper = shallowMount(TmPageSplit, {
      slots: {
        default: "<div>slot</div>"
      }
    })
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
