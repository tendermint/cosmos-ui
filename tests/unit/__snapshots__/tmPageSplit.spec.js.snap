// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmPageSplit.vue has the expected html structure 1`] = `
<div class="tm-page-split">
  <div class="tm-page-split-header"></div>
  <div class="tm-page-split-main">
    <div>
      slot
    </div>
  </div>
</div>
`;
